Vagrantfile

```diff
@@ -6,72 +6,21 @@
 # backwards compatibility). Please don't change it unless you know what
 # you're doing.
 
+$script = <<SCRIPT
```

Comment:
Out of ignorance, what does "<<" do? Do you think it could be documented here? (It's a bit hard to search for.)